packages/legend-graph/src/graph-manager/protocol/pure/v1/engine/execution/V1_ExecutionResult.ts

```diff
@@ -83,6 +87,7 @@ export class V1_JsonExecutionResult extends V1_ExecutionResult {
       _type: primitive(),
       builder: usingModelSchema(V1_ResultBuilder.builderSerialization.schema),
       values: raw(),
+      traceId: optional(primitive()),
```

Comment:
this is `traceId` but above on line 70 it's `traceID`. I think we should standardize it to all be `traceId`.

Reply:
Sry! forgot to clean these code as well. This is not needed as I didn't store traceId inside ExecutionResult anymore.